yb-voyager/cmd/exportData.go

```diff
@@ -552,12 +559,12 @@ func addLeafPartitionsInTableList(tableList []sqlname.NameTuple, ifTableListSet
 		allLeafPartitions := GetAllLeafPartitions(table)
 		prevLengthOfList := len(modifiedTableList)
 		switch true {
-		case len(allLeafPartitions) == 0 && rootTable != table: //leaf partition
+		case len(allLeafPartitions) == 0 && !rootTable.Equals(table): //leaf partition
```

Comment:
Does our `NameTuple.Equals` work if the pointers are different, but the underlying values are the same? 
Is that what prompted you to make this change?

Reply:
Yes, it does the `DeepEqual` internally on the two nametuples.